models/recipe-model.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const recipeSchema = new Schema({
+  title: { type: String, required: true, unique: true },
+  level: {
+    type: String,
+    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]
+  },
+  ingredients: { enum: [] },
```

Comment:
here, you want it to be `{ingredients: {type: Array} }` . An enum is used when there is a multiple choice